src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -218,6 +218,64 @@
   (and (errors? x)
        (some-> x :errors :retryable? boolean)))
 
+(defn parse-args [args clients]
```

Comment:
We hebben ook al een `parse-args-getter`. Ik zou deze `parse-client-info-args` noemen en dan `parse-args-getter` hernoemen naar `parse-getter-args`.